engines/sci/engine/vm.h

```diff
@@ -136,6 +136,21 @@ enum {
 	VAR_PARAM = 3
 };
 
+enum GlobalVar {
+	kEgo            = 0,
+	kGame           = 1,
+	kCurrentRoom    = 2,
+	kSpeed          = 3,  // SCI16
+	kDefaultPlane   = 3,  // SCI32
+	kQuit           = 4,
+	kPlanes         = 10, // SCI32
+	kCurrentRoomNo  = 11,
+	kPreviousRoomNo = 12,
+	kNewRoomNo      = 13,
+	kFastCast       = 84, // SCI16
+	kMessageType    = 90
+};
+
```

Comment:
I have seen that after writing, but it was changed to kGlobalQuit instead of kGlobalVarQuit. And I would also prefer it in case the SCI16/SCI32 exclusive ones were called for example kGlobalVar32Quit or kGlobalVarQuit32.

In other cases we have used the full enum name for the variables too. For example kDebugLevels - that one has kDebugLevelError, kDebugLevelNodes etc.

(in any case this is of course a minor thing and not a big problem - other than that everything looks fine to me (well the SCI16 stuff), excellent work)